apps/api/src/app/access/access.controller.ts

```diff
@@ -169,4 +166,25 @@ export class AccessController {
       );
     }
   }
+
+  @Delete(':id')
```

Comment:
Why has this been moved? Moving code like this makes the review diff much harder to follow.